pkg/applicationmanager/controllers/applicationclone.go

```diff
@@ -119,13 +120,41 @@ func (a *ApplicationCloneController) setDefaults(clone *stork_api.ApplicationClo
 // Make sure the source namespaces exists and create the destination
 // namespace if it doesn't exist
 func (a *ApplicationCloneController) verifyNamespaces(clone *stork_api.ApplicationClone) error {
-	_, err := core.Instance().GetNamespace(clone.Spec.SourceNamespace)
+	ns, err := core.Instance().GetNamespace(clone.Spec.SourceNamespace)
 	if err != nil {
 		return fmt.Errorf("error getting source namespace %v: %v", clone.Spec.SourceNamespace, err)
 	}
-	_, err = core.Instance().CreateNamespace(clone.Spec.DestinationNamespace, nil)
-	if err != nil && !errors.IsAlreadyExists(err) {
-		return fmt.Errorf("error creating destination namespace %v: %v", clone.Spec.DestinationNamespace, err)
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		return err
+	}
+	adminClient, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		return err
+	}
```

Comment:
Don't need this. sched-ops by default picks up config for the local cluster

Reply:
sched-ops apis does not accept annotation field https://github.com/portworx/sched-ops/blob/master/k8s/core/namespaces.go#L49